hs_app_timeseries/models.py

```diff
@@ -1495,333 +1740,310 @@ def _update_timeseriesresultvalues_table_insert(self, con, cur, temp_csv_file, r
-                     "DatasetCode, DatasetTitle, DatasetAbstract) VALUES(?,?,?,?,?,?)"
+                # insert record to SpatialReferences table
+                self.update_spatialreferences_table_insert(con, cur)
 
-        ds_title = self.title.value
-        ds_abstract = self.description.abstract
-        ds_code = self.resource.short_id
-        cur.execute(insert_sql, (1, uuid4().hex, 'Multi-time series', ds_code, ds_title,
-                                 ds_abstract), )
+                # insert record to Sites table
+                self.update_sites_table_insert(con, cur)
 
-    def _update_datatsetsresults_table_insert(self, con, cur):
-        # insert record to DatasetsResults table - first delete any existing records
-        # used for updating a sqlite file that is blank (case of CSV upload)
+                # insert record to Methods table
+                methods_data = self.update_methods_table_insert(con, cur)
 
-        cur.execute("DELETE FROM DatasetsResults")
-        con.commit()
-        insert_sql = "INSERT INTO DatasetsResults (BridgeID, DatasetID, " \
-                     "ResultID) VALUES(?,?,?)"
+                # insert record to Variables table
+                variables_data = self.update_variables_table_insert(con, cur)
 
-        cur.execute("SELECT ResultID FROM Results")
-        results = cur.fetchall()
-        for index, result in enumerate(results):
-            bridge_id = index + 1
-            cur.execute(insert_sql, (bridge_id, 1, result['ResultID']), )
+                # insert record to Units table
+                self.update_units_table_insert(con, cur)
 
-    def _update_utcoffset_related_tables(self, con, cur):
-        # updates Actions, Results, TimeSeriesResultValues tables
-        # used for updating a sqlite file that is not blank and a csv file exists
-        if self.resource.has_csv_file and self.utc_offset.is_dirty:
-            update_sql = "UPDATE Actions SET BeginDateTimeUTCOffset=?, EndDateTimeUTCOffset=?"
-            utc_offset = self.utc_offset.value
-            param_values = (utc_offset, utc_offset)
-            cur.execute(update_sql, param_values)
+                # insert record to ProcessingLevels table
+                pro_levels_data = self.update_processinglevels_table_insert(con, cur)
 
-            update_sql = "UPDATE Results SET ResultDateTimeUTCOffset=?"
-            param_values = (utc_offset,)
-            cur.execute(update_sql, param_values)
+                # insert record to Actions table
+                self.update_actions_table_insert(con, cur, methods_data)
 
-            update_sql = "UPDATE TimeSeriesResultValues SET ValueDateTimeUTCOffset=?"
-            param_values = (utc_offset,)
-            cur.execute(update_sql, param_values)
+                # insert record to People table
+                if not is_file_type:
+                    people_data = self.update_people_table_insert(con, cur)
```

Comment:
Why only inserting record to People table for ts resource type, but not for file type? If there is reason for that, it'd be better to add comments to clarify this for future reference.

Reply:
The creators and contributors can be added only at the resource level and not at the file level.

Reply:
OK. It'd be good to add this clarification as comments for future reference.